sidekick/test/util/cli_runner.dart

```diff
@@ -74,7 +74,7 @@ Future<GlobalSidekickCli> _buildGlobalSidekickCli() async {
   // remove local dependency on sidekick_test, it breaks because of the
   // relative path but can be safely removed because it's just a dev_dependency
   systemDart(
-    ['pub', 'remove', 'sidekick_test', '--offline'],
```

Comment:
No idea why I had to remove `--offline` here - but otherwise this test was failing (only on CI) https://github.com/phntmxyz/sidekick/actions/runs/5201180320/jobs/9381256146